### ISSUE TITLE
fatal error: 'Flutter\Flutter.h' file not found #import <Flutter\Flutter.h> - IOS

### ISSUE THREAD

**Issue by long1eu** [flutter/flutter#79729]
Yet another one of this errors 🗡️ 
```
Launching lib/main.dart on iPhone in debug mode...
Automatically signing iOS for device deployment using specified development team in Xcode project: 447Q6692WZ
Running pod install...
Running Xcode build...
Xcode build done.                                            9.8s
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **


Xcode's output:
↳
    Command CompileSwift failed with a nonzero exit code
    Command CompileSwift failed with a nonzero exit code
    In file included from /Users/long1eu/.pub-cache/hosted/pub.dartlang.org/firebase_crashlytics-1.0.0/ios/Classes/FLTFirebaseCrashlyticsPlugin.m:5:
    /Users/long1eu/.pub-cache/hosted/pub.dartlang.org/firebase_crashlytics-1.0.0/ios/Classes/FLTFirebaseCrashlyticsPlugin.h:8:9: fatal error: 'Flutter/Flutter.h' file not found
    #import <Flutter/Flutter.h>
            ^~~~~~~~~~~~~~~~~~~
    While building module 'firebase_core' imported from /Users/long1eu/.pub-cache/hosted/pub.dartlang.org/firebase_crashlytics-1.0.0/ios/Classes/FLTFirebaseCrashlyticsPlugin.h:12:
    In file included from <module-includes>:1:
    In file included from /Users/long1eu/IdeaProjects/ptch/app/ios/Pods/Target Support Files/firebase_core/firebase_core-umbrella.h:13:
    /Users/long1eu/.pub-cache/hosted/pub.dartlang.org/firebase_core-1.0.2/ios/Classes/FLTFirebaseCorePlugin.h:8:9: fatal error: 'Flutter/Flutter.h' file not found
    #import <Flutter/Flutter.h>
            ^~~~~~~~~~~~~~~~~~~
    1 error generated.
    2 errors generated.
    Command CompileSwift failed with a nonzero exit code
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description
    warning: The Copy Bundle Resources build phase contains this target's Info.plist file '/Users/long1eu/IdeaProjects/ptch/app/ios/Runner/Info_dev.plist'. (in target 'Runner_dev' from project 'Runner')

Could not build the precompiled application for the device.

Error launching application on iPhone.
```

## Steps to Reproduce
`flutter run`

## Logs
```
[✓] Flutter (Channel stable, 2.0.4, on macOS 11.2.3 20D91 darwin-x64, locale en-GB)
    • Flutter version 2.0.4 at /Users/long1eu/SDK/flutter-stable
    • Framework revision b1395592de (2 days ago), 2021-04-01 14:25:01 -0700
    • Engine revision 2dce47073a
    • Dart version 2.12.2

[!] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /Users/long1eu/Library/Android/sdk
    • Platform android-30, build-tools 30.0.3
    • ANDROID_HOME = /Users/long1eu/Library/Android/sdk
    • Java binary at: /Users/long1eu/Library/Application Support/JetBrains/Toolbox/apps/AndroidStudio/ch-0/201.7199119/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
    ✗ Android license status unknown.
      Run `flutter doctor --android-licenses` to accept the SDK licenses.
      See https://flutter.dev/docs/get-started/install/macos#android-setup for more details.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.4, Build version 12D4e
    • CocoaPods version 1.10.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.1)
    • Android Studio at /Users/long1eu/Library/Application Support/JetBrains/Toolbox/apps/AndroidStudio/ch-0/201.7199119/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)

[✓] IntelliJ IDEA Ultimate Edition (version 2020.3.3)
    • IntelliJ at /Users/long1eu/Applications/JetBrains Toolbox/IntelliJ IDEA Ultimate.app
    • Flutter plugin version 55.0.4
    • Dart plugin version 203.7759

[✓] Connected device (3 available)
    • iPhone (mobile) • 00008101-00091C392640001E • ios            • iOS 14.4.2
    • macOS (desktop) • macos                     • darwin-x64     • macOS 11.2.3 20D91 darwin-x64
    • Chrome (web)    • chrome                    • web-javascript • Google Chrome 89.0.4389.114

! Doctor found issues in 1 category.
```

Additional logs attached here 
[debug.txt](https://github.com/flutter/flutter/files/6254068/debug.txt)

or here https://github.com/flutter/flutter/files/6254068/debug.txt

**Comment by long1eu** [comment]
Seems like it work with the following configuration:

```
[✓] Flutter (Channel unknown, 2.0.3, on macOS 11.2.3 20D91 darwin-x64, locale en-GB)
    • Flutter version 2.0.3 at /Users/long1eu/SDK/flutter-2.0.3
    • Framework revision 4d7946a68d (2 weeks ago), 2021-03-18 17:24:33 -0700
    • Engine revision 3459eb2436
    • Dart version 2.12.2

[!] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /Users/long1eu/Library/Android/sdk
    • Platform android-30, build-tools 30.0.3
    • ANDROID_HOME = /Users/long1eu/Library/Android/sdk
    • Java binary at: /Users/long1eu/Library/Application Support/JetBrains/Toolbox/apps/AndroidStudio/ch-0/201.7199119/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
    ✗ Android license status unknown.
      Run `flutter doctor --android-licenses` to accept the SDK licenses.
      See https://flutter.dev/docs/get-started/install/macos#android-setup for more details.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.4, Build version 12D4e
    • CocoaPods version 1.10.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.1)
    • Android Studio at /Users/long1eu/Library/Application Support/JetBrains/Toolbox/apps/AndroidStudio/ch-0/201.7199119/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)

[✓] IntelliJ IDEA Ultimate Edition (version 2020.3.3)
    • IntelliJ at /Users/long1eu/Applications/JetBrains Toolbox/IntelliJ IDEA Ultimate.app
    • Flutter plugin version 55.0.4
    • Dart plugin version 203.7759

[✓] Connected device (3 available)
    • iPhone (mobile) • 00008101-00091C392640001E • ios            • iOS 14.4.2
    • macOS (desktop) • macos                     • darwin-x64     • macOS 11.2.3 20D91 darwin-x64
    • Chrome (web)    • chrome                    • web-javascript • Google Chrome 89.0.4389.114

! Doctor found issues in 1 category.

```

**Comment by fareesh** [comment]
+1

I've tried flutter clean, pod disintegrate/update, and all the steps suggested in a previous version of this issue

**Comment by daniellampl** [comment]
Obviously there is a bug in Version 2.0.4 which causes this problem. Downgrading seems to do the job 🎉

**Comment by isfundiyor** [comment]
I've downgraded to flutter 2.0.3, but it didn't work, any other solution?

**Comment by nt4f04uNd** [comment]
duplicate of #79575
for summarized explanations see this https://github.com/flutter/flutter/issues/79722#issuecomment-813046984

**Comment by TahaTesser** [comment]
Hi @nt4f04uNd 
This doesn't seem like it's related to Flutter itself https://github.com/flutter/flutter/issues/79722#issuecomment-813046984
Closing as duplicate of https://github.com/flutter/flutter/issues/79575
If you disagree, please write in the comments and I will reopen it.
Thank you
